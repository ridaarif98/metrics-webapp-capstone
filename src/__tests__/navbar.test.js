import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../compoenets/navbar';

describe('Test to see that all objects render correctly.', () => {
  test('Check if Home renders.', () => {
    const tree = renderer
      .create(
        <Router>
          <NavBar />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
