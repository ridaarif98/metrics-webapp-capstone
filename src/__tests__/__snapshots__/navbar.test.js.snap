// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to see that all objects render correctly. Check if Home renders. 1`] = `
<nav
  className="navBar"
>
  <ul>
    <li>
      <a
        className="navLink"
        href="/"
        onClick={[Function]}
      >
        &lt;
      </a>
    </li>
  </ul>
  <h3>
    Covid-19 metrics
  </h3>
  <div>
    <i
      className="fa fa-microphone"
    />
    <i
      className="fa fa-gear"
    />
  </div>
</nav>
`;
